pom.xml

```diff
@@ -142,8 +142,8 @@
         <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ss</maven.build.timestamp.format>
         <timestamp>${maven.build.timestamp}</timestamp>
 
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
changing the `source` we will not be able to use Java 17 features in the codebase.
is there any issue with setting only the `target` ?

Reply:
Not allowed by the java compiler, sadly.